mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -298,7 +306,23 @@ class Completions(
                 ).forall(visit),
         )
         Some(search.search(query, buildTargetIdentifier, visitor))
-      case _ => None
+      case CompletionKind.Members if query.nonEmpty =>
```

Comment:
Nice rework in general:+1: 

It would be ideal if it were allowed to search even with an empty query.
Then, user will be able do discover extension method by autocomplete

Reply:
Thank you for your comment!
Right, it would be really helpful, I remember I guarded by `query.nonEmpty` because something has failed if a query is empty, but we can fix it and make them available with empty query 👍